marimo/_runtime/runner/cell_runner.py

```diff
@@ -431,6 +433,33 @@ async def run(self, cell_id: CellId_t) -> RunResult:
             ):
                 self.missing_packages = True
 
+                # If the user has a library and a file with the same name
+                # we should inform that this is a conflict.
+                if isinstance(unwrapped_exception, ModuleNotFoundError):
+                    try:
+                        module_name = getattr(unwrapped_exception, "name", "")
+                        module_name = module_name.split(".")[0]
+
+                        python_files = [
+                            file.split(".")[0]
+                            for file in os.listdir()
+                            if file.endswith(".py")
+                        ]
+                        if (
```

Comment:
rather than listing the directory, could we do `if Path(f"{module_name}.py").exists() and DependencyManager.has(module_name):`